templates/angular/igx-ts/autocomplete/autocomplete-custom/index.ts

```diff
@@ -6,7 +6,7 @@ class IgxAutocompleteTemplate extends IgniteUIForAngularTemplate {
 		this.components = ["Autocomplete"];
 		this.controlGroup = "Data Entry & Display";
 		this.listInComponentTemplates = true;
-		this.id = "autocomplete";
+		this.id = "extended-autocomplete";
```

Comment:
I think since this is the "Simple" Autocomplete, we can leave the id to be just `"autocomplete"`. It's a bit more intuitive and we might have it somewhere in the public docs, so this way, we'll avoid a potential breaking change.